CHANGELOG.md

```diff
@@ -118,6 +119,7 @@ Cosmwasm bindings have been added, to make swapping and creating these new token
 * [#1656](https://github.com/osmosis-labs/osmosis/pull/1656) Change camelCase to snake_case in proto.
 * [#1632](https://github.com/osmosis-labs/osmosis/pull/1632) augment SuperfluidDelegationsByDelegator query, return osmo equivilent is staked via superfluid
 * [#1723](https://github.com/osmosis-labs/osmosis/pull/1723) fix number of LP shares returned from stableswap pool
+* [#1539](https://github.com/osmosis-labs/osmosis/pull/1539) Superfluid: Combine superfluid and staking query on querying delegation by delegator
```

Comment:
This is the v9.0.0 section! A single entry above should suffice!

Reply:
deleted